WPMedia/fusion-news-theme-blocks
# Tmedia 402 logged out nav cta

## Description
As a user, I can click “Sign Up” or “Log In” CTAs from the navigation.

## Jira Ticket
- [TMEDIA-402](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-402&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
- [x] Sign up URL is a Custom field with a default of account/signup
- [x] Log In URL is a Custom field with a default of account/login
- [x] Clicking Sign Up nav CTA on Desktop takes you to sign up URL
- [x] Clicking Log In nav CTA on Desktop takes you to log in URL
- [x] Single Button on Mobile w/dropdown on click
- [x] Primary font is used -> see reusable button component
- [x] Language keys are used for displaying words -> "Sign Up" and "Log In" are in the intl files
- [x] Accessibility is to be taken into account
E.g. Button when a button, link when a user is being taken to a page -> using a link tag for the sign up links and the "as" to modify the styled component

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-402-logged-out-nav-cta`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. Create page with header nav chain
4. Add header nav action to desktop and mobile custom placements 
5. Change screen and see both rendered as expected on published page with "Sign Up" and "Log In" on desktop and an account button with an on-click dropdown on mobile 
6. With custom fields you should be able to change the urls to the page

https://zpl.io/2E85xAn - zeplin docs 

## Effect Of Changes
### Before
_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After
_Example: When I clicked the search button, the button turned green._

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
